shed/tests/test_streams.py

```diff
@@ -1043,10 +1062,10 @@ def test_curate_streams():
     doc4_curated2 = s.curate_streams(doc4_curated, True)
     doc5_curated2 = s.curate_streams(doc5_curated, True)
 
-    assert doc1_curated == ('start', (None, ))
-    assert doc1_curated2 == ('start', None, )
+    assert doc1_curated == ('start', (None,))
+    assert doc1_curated2 == ('start', None,)
```

Comment:
Internally everything is converted into a tuple within a tuple. This way we have a uniform interface to the data. On the way out we remove length one tuples and replace them with their contents.